apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -657,17 +527,12 @@ impl BridgeQueryPlanner {
         if let Some((unauthorized_paths, new_doc)) = filter_res {
             key.filtered_query = new_doc.to_string();
             let executable = new_doc
-                .to_executable(&self.schema.api_schema().definitions)
+                .to_executable_validate(&self.schema.api_schema().definitions)
                 // Assume transformation creates a valid document: ignore conversion errors
-                .unwrap_or_else(|invalid| invalid.partial);
+                .unwrap();
```

Comment:
Unwrap here, is this safe?

Reply:
we assume that the operations we generate are valid, but just in case I'll raise an error here

Reply:
bf7669828